src/main/app/case/CaseApi.ts

```diff
@@ -71,16 +83,29 @@ export class CaseApi {
     }
   }
 
-  public async triggerEvent(caseId: string, userData: Partial<Case>, eventName: string): Promise<CaseWithId> {
+  public async triggerEvent({
+    caseId,
+    data = {},
+    eventName = CITIZEN_UPDATE,
```

Comment:
I don't think we should default value this. In think I need to understand why you're changing this method